### PR TITLE
Drop for TimeoutExecutor on win10

### PR DESCRIPTION
**Pull request by tokatoka**
I should delete the timer_queue object when the TimeoutExecutor goes out of the scope.

Also I have to add several lines to `executor/command.rs` because it is emitting 3 warnings on compiling on windows.

### CODE REVIEW

File: libafl/src/executors/command.rs [hunk 1]
```diff
@@ -19,6 +20,12 @@ pub struct CommandExecutor<EM, I, S, Z, T, OT> {
     phantom: PhantomData<(EM, I, S, Z)>,
 }
 
+impl<EM, I, S, Z, T, OT> CommandExecutor<EM, I, S, Z, T, OT> {
```
Comment (domenukk) [inline libafl/src/executors/command.rs:23]: Unrelated, but some when are these not sorted alphabetically? 🤔

Comment (tokatoka) [reply libafl/src/executors/command.rs:23]: ok I can reorder these

Comment (tokatoka) [reply libafl/src/executors/command.rs:23]: oops it got already merged 😓

Comment (domenukk) [reply libafl/src/executors/command.rs:23]: Open a new PR for the reorder if you want :P

**Pull request merged**